### DISCUSSION TITLE
limit_execution time+8

### DISCUSSION THREAD

**Discussion by Pa55w0rd** [jertel/elastalert2#843] [Q&A]
[root@localhost elastalert]# date
Fri May 13 14:54:06 CST 2022

rule:
limit_execution: "30 17 * * *"

run:
1 rules loaded
INFO:elastalert:Starting up
INFO:elastalert:Disabled rules are: []
INFO:elastalert:Sleeping for 299.999865 seconds
INFO:elastalert:Pausing test until next run at 2022-05-14 01:30 CST

2022-05-13 17:30    -- > 2022-05-14 01:30
+8

**Comment by nsano-rururu** [comment]
https://github.com/Yelp/elastalert/issues/492
